service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -412,10 +412,12 @@ public ApplicationsBundle getApplicationBundleByUUID(UUID elementId) {
     private final Integer orderMonths;
     private final InterimEndDate interimEndDate;
     private final Selector childSelector;
+    private final Selector othersSelector;
```

Comment:
that goes in CaseData because it's generic? or in ManageOrdersEventData. If that's true, maybe othersSelector etc should not go in manageOrdersFields.json